lib/passport-saml/saml.js

```diff
@@ -778,6 +778,7 @@ function processValidlySignedSamlLogout(self, doc, callback) {
 }
 
 SAML.prototype.hasValidSignatureForRedirect = function (samlMessageType, container) {
+  var self = this;
```

Comment:
`self = this` is not necessary if arrow functions are used, which are supported on all Node versions that we support.